history.rst

```diff
@@ -5,6 +5,24 @@
 OMERO version history
 =====================
 
+5.6.2 (July 2020)
+-----------------
+
+This release adds installation documentation for server and web on
+CentOS 8 and Ubuntu 20.04, and it removes installation instructions
+for Ubuntu 16.04.
```

Comment:
Are we selling this as the main feature of the release or are there other high level improvements worth adding? As such it feels like there is little benefit to upgrading a 5.6.x server on a supported platform for instance.

For Ubuntu 16.04, should mention drop support?

Reply:
Without replicating the content of the changelogs of each "component", this history file is becoming less verbose.

Reply:
Understood. So is the announcement the best reference place to get an overview of all end-users features associated with each release